.github/ISSUE_TEMPLATE/bug-report.md

```
@@ -27,3 +28,9 @@ A clear and concise description of how the code performed w.r.t expectations.
 If applicable, add screenshots to help explain your problem.
 
 **Additional details**
+Add any other context or screenshots about the feature request here.
+
+**Potential internship candidates**
+
+Please read this if you are planning to apply for a Palisadoes Foundation internship 
+- https://github.com/PalisadoesFoundation/talawa/issues/359
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)